src/firefly/js/charts/ChartUtil.js

```diff
@@ -504,6 +504,10 @@ function tablesourcesEqual(newTS, oldTS) {
 
 
 function updateChartData(chartId, traceNum, tablesource, action={}) {
+    // Histogram doesn't update on a table sort event.
+    if (getChartData(chartId).fireflyData[0].dataType === 'fireflyHistogram'){
+        if (action.payload && action.payload.invokedBy === 'table.sort'){return;}
+    }
```

Comment:
There should be `traceNum` instead of `0`. 
Also you might want to move it to "else" block and rewrite it a bit to capture other non-scatter trace types, since the order of table rows only matters for scatter traces:
```
        if (action.type === TABLE_LOADED && action.payload.invokedBy === TABLE_SORT) {
            const {data} = getChartData(chartId);
            const traceType = get(data, [traceNum, 'type'], 'scatter');
            if (!traceType.includes('scatter')) return;
        }
```

Reply:
Done, thanks!